core/extension/gdextension_interface.h

```diff
@@ -2126,6 +2171,10 @@ typedef void (*GDExtensionInterfaceRefSetObject)(GDExtensionRefPtr p_ref, GDExte
  */
 typedef GDExtensionScriptInstancePtr (*GDExtensionInterfaceScriptInstanceCreate)(const GDExtensionScriptInstanceInfo *p_info, GDExtensionScriptInstanceDataPtr p_instance_data);
 
+#ifndef DISABLE_DEPRECATED
+typedef GDExtensionScriptInstancePtr (*GDExtensionInterfaceScriptInstanceCreate1)(const GDExtensionScriptInstanceInfo1 *p_info, GDExtensionScriptInstanceDataPtr p_instance_data);
+#endif // DISABLE_DEPRECATED
```

Comment:
We don't put `#ifdef`s like this in `gdextension_interface.h`. But per my previous notes, you'll be able to remove all this and just take advantage of the `GDExtensionScriptInstanceInfo2` and `GDExtensionInterfaceScriptInstanceCreate2` that have already been added in the master branch